File: q0867/q0867.py
class Solution(object):
    def transpose(self, matrix):
        """
        :type matrix: List[List[int]]
        :rtype: List[List[int]]
        """
        m, n = len(matrix), len(matrix[0])

        transposedMatrix = [[None] * m for _ in range(n)]

        for i in range(m):
            for j in range(n): 
                transposedMatrix[j][i] = matrix[i][j]
        
        return transposedMatrix
        

File: q0867/q0867.java
class Solution {
    public int[][] transpose(int[][] matrix) {
        int m = matrix.length;
        int n = matrix[0].length;

        int[][] transposedMatrix = new int[n][m];

        for (int i = 0; i < m; i++) {
            for (int j = 0; j < n; j++) {
                transposedMatrix[j][i] = matrix[i][j];
            }
        }

        return transposedMatrix;
    }
}